content/spaces/kind-quickstart.md

```diff
@@ -114,35 +110,26 @@ You are ready to [create your first managed control plane](#create-your-first-ma
 
 With your kubeconfig pointed at the Kubernetes cluster where you installed the Upbound Space, create a managed control plane:
 
-```yaml
-cat <<EOF | kubectl apply -f -
-apiVersion: spaces.upbound.io/v1alpha1
-kind: ControlPlane
-metadata:
-  name: ctp1
-spec:
-  writeConnectionSecretToRef:
-    name: kubeconfig-ctp1
-    namespace: default
-EOF
+```shell
+up ctp create controlplane1
 ```
 
 The first managed control plane you create in a Space takes around 5 minutes to get into a `condition=READY` state. Wait until it's ready using the following command:
 
-```bash
-kubectl wait controlplane ctp1 --for condition=Ready=True --timeout=360s
+```shell
```

Comment:
We should remove this command and replace it perhaps with `up ctp list`, but we also need to chnage the wording if we do that since that command doesn't have a wait loop, but it does report the status of the ctp.